src/templates/pages/example/full-screen-navigation/code.js

```diff
@@ -1,14 +1,14 @@
 import { h } from 'preact';
 
 const Code = () => <nav aria-label={'Primary navigation'}>
-    <button type="button" class="full-screen-nav__btn js-full-screen-nav__toggle" aria-label="Open menu" aria-controls="full-screen-navigation" aria-expanded="false">
+    <button type="button" class="full-screen-nav__btn js-full-screen-nav__toggle" aria-label="Open menu" aria-controls="full-screen-navigation" aria-expanded="false" aria-haspopup="menu">
         <svg focusable="false" class="full-screen-nav__btn-icon" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
             <path d="M0 0h24v24H0z" fill="none" />
             <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z" />
         </svg>
         menu
     </button>
-    <div id="full-screen-navigation" class="js-full-screen-nav full-screen-nav" data-toggle="js-full-screen-nav__toggle">
+    <div id="full-screen-navigation" class="js-full-screen-nav full-screen-nav" data-toggle="js-full-screen-nav__toggle" aria-role="menu">
         <ul class="wrap full-screen-nav__list">

```

Comment:
I think this should be 'role="menu"'.

With that there are a few criteria that menus should meet with subitems with 'role="menuitem"', and keyboard interactions (escape to close, down/right arrows to navigate) that we don't have in this implementation and would probably be a separate component?

Reply:
^ that was flagged by accessibility tools in Chrome, and details in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role#example_1_navigation_menu

Reply:
Oops - of course. Fixed the attribute!  Maybe we should raise another ticket to look at the menu's AX separately?

Reply:
Yeah, I think that might be best, give ourselves a chance to review and test if we want (or need) to go all-in with menu role.

Reply:
Cool, added here:  [https://github.com/stormid/ui-patterns/issues/82](https://github.com/stormid/ui-patterns/issues/82)

Reply:
Should we remove role=menu for now pending the other investigation? A role of menu without menuitems (links with role=menuitem) is flagged as invalid in accessibility tools.

Reply:
Got you, yes - agreed.  I've removed the menu role and the aria-haspop="menu" for now (as that's not valid without the explicit role) and concentrated on only the dialogs for now.   I'll add a point to the other ticket to remember to revisit this.

I'd also missed running the Jest tests on this one, so the latest push fixes a couple of errors there.